testsuite/.rubocop_todo.yml

```diff
@@ -432,10 +432,11 @@ Style/IfUnlessModifier:
     - 'features/step_definitions/retail_steps.rb'
     - 'features/step_definitions/salt_steps.rb'
     - 'features/step_definitions/setup_steps.rb'
+    - 'features/step_definitions/vm_steps.rb'
```

Comment:
Same, I would disable this Cop for all the files.

Reply:
This is slightly more complex to do. I would also postpone this to another future PR.